MCMC/simulation.py

```diff
@@ -1,9 +1,25 @@
-import numpy as np
 import itertools
 import math
 
+import numpy as np
+
+from utils import inverse_distance_energy
+
+
 class Simulation:
-    def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5):
+    def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5,
```

Comment:
IMO, it is most simple to have a default radius of 1, then L is the count of radii. So, in the case of default r=1.0 and L=50; the box lengths are 25 circles long.  Technically this is no different than what you are doing it, but let's us start with easier numbers to work with.